src/Controller/Component/AuthComponent.php

```diff
@@ -349,7 +349,7 @@ protected function _unauthenticated(Controller $controller)
         }
 
         if (!empty($this->_config['ajaxLogin'])) {
-            $controller->viewPath = 'Element';
+            $controller->getview()->viewPath = 'Element';
```

Comment:
I wonder if we should add a getter method for that property instead of setting it directly